pkg/crc/machine/disks.go

```diff
@@ -125,6 +133,14 @@ func getrootPartition(sshRunner *crcssh.Runner, preset crcPreset.Preset) (string
 	if preset == crcPreset.Microshift {
 		query = "-t TYPE=LVM2_member"
 	}
+	return getDeviceID(sshRunner, query)
+}
+
+func getTopolvmPartition(sshRunner *crcssh.Runner) (string, error) {
+	return getDeviceID(sshRunner, "-t PARTLABEL=topolvm")
+}
+
+func getDeviceID(sshRunner *crcssh.Runner, query string) (string, error) {
```

Comment:
I find the api to this function a bit odd, this `query` string is quite magic. Might be better to name it `runBlkidQuery` or such?